INFO - 2023-05-03 20:10:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 20:15:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-05-03 20:15:58 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'usuarios' in C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'usuarios', 3306, '', 0)
#1 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#4 C:\xampp\htdocs\proyectoFundamentos\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectoFundamentos\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectoFundamentos\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectoFundamentos\app\Controllers\Home.php(81): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(934): App\Controllers\Home->guardar()
#9 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#10 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectoFundamentos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyectoFundamentos\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'usuarios' in C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pe...', Array, false)
#3 C:\xampp\htdocs\proyectoFundamentos\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\proyectoFundamentos\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\proyectoFundamentos\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\proyectoFundamentos\app\Controllers\Home.php(81): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(934): App\Controllers\Home->guardar()
#8 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#9 C:\xampp\htdocs\proyectoFundamentos\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyectoFundamentos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\proyectoFundamentos\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-05-03 20:15:58 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'usuarios'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `personas` (`nombre`, `apellido`, `telefono`, `correo`, `usuario`, `password`) VALUES (:nombre:, :apellido:, :telefono:, :correo:, :usuario:, :password:)', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Home.php(81): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->guardar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\proyectoFundamentos\\public\\index.php')
ERROR - 2023-05-03 20:19:33 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'usuarios' in C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'usuarios', 3306, '', 0)
#1 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\proyectoFundamentos\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\proyectoFundamentos\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\proyectoFundamentos\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\proyectoFundamentos\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\proyectoFundamentos\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\proyectoFundamentos\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'usuarios' in C:\xampp\htdocs\proyectoFundamentos\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\proyectoFundamentos\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\proyectoFundamentos\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\proyectoFundamentos\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\proyectoFundamentos\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\proyectoFundamentos\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\proyectoFundamentos\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\proyectoFundamentos\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
INFO - 2023-05-03 20:20:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 20:20:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 20:29:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 20:30:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 20:30:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:51:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:51:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:51:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:53:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:53:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:54:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:54:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:54:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:55:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:55:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:56:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-05-03 21:56:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
